### PR TITLE
"Other" V2 Idlers Mod

### PR DESCRIPTION
**Pull request by selliott79**
* [ X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ X] Submitted STL files are oriented and scaled properly for printing.
  * [ X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
      "Other" V2.4 Idler Mod
#### How was it tested? 
      I have run this mod for 50+ hrs. on my V2.4 without issue. Others are currently printing it and planning to implement it on their machines. 
#### Any background context you want to provide?
         No
#### Screenshots (if appropriate)
         Built-in to the readme file
#### Further notes
         No

### CODE REVIEW

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# "Other" V2 Idlers
+ Alternative Idlers that allow the usage of "Full-Figured" toolheads without the 10mm loss of travel at the front corners of the bed.
+
+These idlers were developed with the goal to allow the usage of wider toolheads with the Voron V2, better align the tensioning screw to the centerline of the idler shaft, add guides to the parts to allow them to interlock and provide a robust tensioning system, all while streamlining the process of making fine adjustments to tensioning. 
+This design iteration has been running on my machine for 50+ hrs with no issues or print quality degradation.  
+
+The idler design is ambidextrous to allow simplified batch printing. All parts are converted to stl in optimal printing orientation with quantities specified per Voron standards: 
+
+![image](https://user-images.githubusercontent.com/65414930/123886695-a1619680-d915-11eb-8a33-212a3cb1b8ef.png)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:9]: Please upload all images used in this README into a directory under your mod, e.g. `printer_mods/Phalanx/Other-V2-Idlers/images/`, and then update this README to reference those uploaded images.  For example, if this image is uploaded as `images/naming_standards.png`, then reference it in the README as

    Naming standards

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+# Print Settings:
+All parts should be printed to standard Voron print settings:
+
+![image](https://user-images.githubusercontent.com/65414930/123886729-b63e2a00-d915-11eb-81b5-52c1082b56c9.png)
+
+Note: If your printer is not properly tuned i.e. printing dimensionally-accurate parts free from functional defects, the slide mechanism WILL NOT work proerly. 
+
+# BOM:
+2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source.
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:20]: Can you convert this a markdown list, so it displays one per line?  Right now it looks like this:

Screen Shot 2021-07-03 at 14 43 11

You can make it a list by preceding each line with a `- `, e.g:

```
# BOM
- 2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source.
- 4 ea. F695 bearings
```

which would look like:

# BOM
- 2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source.
- 4 ea. F695 bearings

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+![image](https://user-images.githubusercontent.com/65414930/122688376-7ffbfe80-d1e1-11eb-84d0-311bcf6c26bf.png)
+
+
+
+![image](https://user-images.githubusercontent.com/65414930/122688437-c6515d80-d1e1-11eb-84bf-6041ab9b4e93.png)
+
+
+![20210619_102801](https://user-images.githubusercontent.com/65414930/122848149-98961280-d2ce-11eb-99c0-5b0b7387a849.jpg)
+Be sure to fully insert the m3 heat sink so that it doesn't interfere with the bearings!
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:48]: Change `heat sink` to `heat set insert`.

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+![image](https://user-images.githubusercontent.com/65414930/123886729-b63e2a00-d915-11eb-81b5-52c1082b56c9.png)
+
+Note: If your printer is not properly tuned i.e. printing dimensionally-accurate parts free from functional defects, the slide mechanism WILL NOT work proerly. 
+
+# BOM:
+2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source. 
+4 ea. F695 bearings
+4 ea. 0.5mm thick 5mm shims
+2 ea. M3x35mm length SHCS - Stock 40mm may be run if not using the bolted door hinge mod.
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:23]: Add a note here that the screw must be fully threaded, as some long screws linked from the sourcing guide have a shoulder at the top and are not fully threaded.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please add your mod to `printer_mods/README.md` in alphabetical order.

For `STL/[a]_Idler_Carrierx2.stl`, that print orientation is probably fine, give that you've been running the idlers for a while like that, but since the tension applied to them by the tensioning screw essentially stretches it in the Z direction, would it make sense to redesign it so it prints lying on its side instead?

**Comment by selliott79** [comment]
Hi Kilo,
   I appreciate you taking the time to review my mod pull request and
provide feedback. I'll try and get changes squared away by the weekend.

Thanks again!
Phalanx

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- 2 ea. M3 nut
+- 2 ea. Voron - standard heat set
+- 2 ea. M3 x 16 SHCS
+
+These components are being offered as a kit by https://formosissima.com/products/other-v2-idlers-kit
+
+This mod also requires that you run https://github.com/VoronDesign/Voron-2/blob/Voron2.4/STLs/VORON2.4/Gantry/%5Ba%5D_z_belt_clip_lower_x4.stl
+on both top and bottom as belt clips, so that the design is not handed. 
+
+![Idler comparison](images/Idler comparison.png)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:29]: once you remove the space from that image filename (and it's also a jpg, not png), update this to reflect the new name, i.e.

    Idler comparison

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+![image](./Phalanx/Other-V2-Idlers/Images/20210619_204308.jpg)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:30]: You don't have an image with this name in your Images directory.  The relative reference is also incorrect, it should be more like (if that image actually existed in the Images directory):

    image

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+"Other" Idlers Thickness vs. stock V2.4 idlers
+
+
+![image](https://user-images.githubusercontent.com/65414930/122688376-7ffbfe80-d1e1-11eb-84d0-311bcf6c26bf.png)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:34]: Please upload this image to the Images directory and update this reference to point to it.

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+
+
+![image](https://user-images.githubusercontent.com/65414930/122688437-c6515d80-d1e1-11eb-84bf-6041ab9b4e93.png)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:38]: Please upload this image to the Images directory and update this reference to point to it.

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+
+![20210619_102801](printer_mods/Phalanx/Other-V2-Idlers/Images/20210619_102801.jpg)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:41]: You don't have an image with this name in your Images directory.  The relative reference is also incorrect, it should be more like (if that image actually existed in the Images directory):

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+Be sure to fully insert the m3 heat set insert so that it doesn't interfere with the bearings!
+
+![20210619_114935](https://user-images.githubusercontent.com/65414930/122848219-b95e6800-d2ce-11eb-8364-58fcbf4ed25d.jpg)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:44]: Please upload this image to the Images directory and update this reference to point to it.

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+
+![20210620_155754](https://user-images.githubusercontent.com/65414930/122848232-bd8a8580-d2ce-11eb-8203-3d85482990ca.jpg)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:47]: Please upload this image to the Images directory and update this reference to point to it.

File: printer_mods/Phalanx/Other-V2-Idlers/README.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+![20210619_213153](https://user-images.githubusercontent.com/65414930/122848297-e14dcb80-d2ce-11eb-9942-082bb1cd4e2e.jpg)
```
Comment (KiloQubit) [inline printer_mods/Phalanx/Other-V2-Idlers/README.md:49]: Please upload this image to the Images directory and update this reference to point to it.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your updates!

Please remove the space from `Images/Idler comparison.jpg` filename.

It also looks like you still have a lot of images in the README that are not in the Images directory.

**Comment by selliott79** [comment]
Hi, I updated as you asked above but for whatever reason it shows a broken
link to the image in the images folder. I am still REALLY new to git, could
you please advise on where I am going wrong?

**Comment by KiloQubit** [comment]
> Hi, I updated as you asked above but for whatever reason it shows a broken link to the image in the images folder. I am still REALLY new to git, could you please advise on where I am going wrong?
> […](#)

Your directory is called `Images`, but you're referencing it as `images` in the README.  Case matters.  Change your references from `images` to `Images`.

**Comment by selliott79** [comment]
I see, thanks for the pointer! They should all be working now.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Comment by selliott79** [comment]
Awesome, thank you for the help!

**Pull request merged**